src/index.html

```diff
@@ -54,6 +54,14 @@
                         <div class="form-group">
                             <button id='refresh-data' class="btn btn-primary" type="button">Refresh</button>
                         </div>
+                        <div class="toast" style="display:none">
```

Comment:
styles in the css, please don't put them inline

Reply:
Fixed